chaoslib/pumba/network_chaos/pumba_netem_job.j2

```diff
@@ -2,15 +2,19 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: pumba-netem-{{ run_id }}
-  labels: 
+  labels:
+{% if c_engine is defined and c_engine != '' %}  
```

Comment:
If only one label is present then put label inside if condition like:
{% if c_engine is defined and c_engine != '' %}  
labels:
chaosUID: {{ chaos_uid }}
{% endif %}

If multiple labels are present then you can use :
labels:
{% if c_engine is defined and c_engine != '' %}  
chaosUID: {{ chaos_uid }}
{% endif %}

Reply:
other one is mandatory doesn't depend on anyone so keeping it away from if - else